controllers/flowcollector_controller.go

```diff
@@ -253,6 +252,11 @@ func (r *FlowCollectorReconciler) setupDiscovery(ctx context.Context, mgr ctrl.M
 	r.availableAPIs = apis
 	if apis.HasConsole() {
 		builder.Owns(&osv1alpha1.ConsolePlugin{})
+	} else {
+		log.Info("Console not detected: the console plugin is not available")
+	}
+	if !apis.HasCNO() {
+		log.Info("CNO not detected: using ovnKubernetes config and reconciler")
 	}
```

Comment:
Is the intention here only to log a message?

Reply:
yes, I find it useful while checking the logs :)